src/sif/bicyclecost.cc

```diff
@@ -750,7 +758,7 @@ Cost BicycleCost::TransitionCost(const baldr::DirectedEdge* edge,
   penalty *= (bike_accom * avoid_roads) + use_roads_;
 
   // Return cost (time and penalty)
-  c.cost += (seconds * (turn_stress + 1.0f)) + penalty;
+  c.cost += shortest_ ? 0.f : (seconds * (turn_stress + 1.0f)) + penalty;
```

Comment:
this is also not quite right. above the `base_transition_cost` function will have set the cost to something other than 0. but if we are doing shortest we need it to always be 0. actually thats probably a good thing to check in the funciton and set to 0. let me do that

Reply:
haha me from 2 months ago tried to do that but made a mistake... let me fix my mistake :smile: